recipes/couchdb/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "couchdb" %}
+{% set version = "3.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://couchdb.neighbourhood.ie/downloads/{{ version }}/win/apache-couchdb-{{ version }}.msi  # [win]
+    sha256: f7d354d5ede02d2af748f8d3230bd46b9c3e70e7cd6430ddf2095916d260060e  # [win]
+  - url: https://archive.apache.org/dist/couchdb/source/{{ version }}/apache-couchdb-{{ version }}.tar.gz  # [unix]
+    sha256: 69c9fd6f80133557f68a02e92dda72a4fd646d646f429f45bb8329a30f82f20e  # [unix]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [unix]
+    - {{ compiler('cxx') }}  # [unix]
+    - make  # [unix]
+  host:
+    - libffi  # [unix]
+    - nspr  # [unix]
+    - erlang  # [unix]
+    - icu  # [unix]
+    - spidermonkey 91.*  # [unix]
+  run:
+    - openssl  # [unix]
+    - icu  # [unix]
+    - libffi  # [unix]
+    - nspr  # [unix]
+    - ncurses  # [unix]
+    - libzlib  # [unix]
+    - spidermonkey 91.*  # [unix]
+
+test:
+  commands:
+    - couchjs -V
```

Comment:
This version print [claims spidermonkey 86](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=507484&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=5335), but we are using spidermonkey 91. Is this our fault or is this text hardcoded? Or we aren't using the spidermonkey that we built on conda-forge?